Services/PetService.cs

```diff
@@ -1,53 +1,239 @@
-using bootcamp_api.Models;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.Design;
+using System.Drawing;
+using System.Linq;
+using System.Xml.Linq;
+using Domain;
+using bootcamp_api.Data;
+using bootcamp_api.Exceptions;
+using Microsoft.EntityFrameworkCore;
 
-namespace bootcamp_api.Services;
-
-public static class PetService
```

Comment:
Move all your mapping from the controllers to the services so that they return DTOs instead database models.
Your Service acts as the translation layer between your DB and your Controller.  
https://www.thereformedprogrammer.net/six-ways-to-build-better-entity-framework-core-and-ef6-applications/#the-service-layer-as-an-adapter

This is important to my comment below. Instead of returning list of Pets it will allow you to return a list of much smaller objects that you need to display when you do a search.

Reply:
Mapping is now in the service layer.